jetty-osgi/test-jetty-osgi/pom.xml

```diff
@@ -409,18 +428,6 @@
       <artifactId>jetty-test-helper</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
```

Comment:
Keep in mind that `slf4j-api` is `<scope>compile</scope>` by default, that means this change made it go from `test` to `compile` for this project.

Reply:
As this project is a test project, that's fine.